com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -653,6 +654,8 @@ private void OnEnable()
 
         private void OnDestroy()
         {
+            OnNetworkLoopSystemRemove();
+            //NSS: This is ok to leave this check here
             rpcQueueContainer?.OnExiting();
```

Comment:
`?.` vs `.` ? o.o

Reply:
I am going to leave that check there.  It is always a good idea to check things as you are destroying a singleton class.
"just in case"